kube-client/src/client/body.rs

```diff
@@ -44,6 +44,11 @@ impl Body {
         Body::new(Kind::Wrap(body.map_err(Into::into).boxed_unsync()))
     }
 
+    /// Collect all the pieces of the body
```

Comment:
I know this is really nitpicky but we could also phrase this as: 

```suggestion
    /// Collect all the data frames and trailers of the request body
```

to be more consistent with [upstream comments](https://docs.rs/http-body-util/latest/http_body_util/trait.BodyExt.html#method.collect); if it doesn't make it more unintuitive for users that is.